app/assets/javascripts/scoring.js

```diff
@@ -16,4 +17,9 @@ $(document).ready(function(){
       if(max && value === parseInt(max)) $(this).prop('disabled', true);
     }
   });
+
+  $('.scoring').on('click', '.pseudo-checkbox', function(){
```

Comment:
`click` is probably the only event that matters, right? I have this vague sense that I've been burned before by needing to handle some other (maybe mobile?) case for checkboxes...

Reply:
So, first off, this isn't actually a checkbox (which, I think _would_ make things more complicated). It used to be that there was (on most mobile browsers) a 300ms delay between when the user touched the screen and when the click event fired. This was so that the browser could determine whether you were actually double-tapping-to-zoom and not clicking on something. One work-around to that problem was to listen for `touchstart`, but them you also have to deal with the fact that `click` happens _too_. There are some helper libraries out there that essentially do:

* Listen for `touchstart` or `touchend`
* Prevent default
* Trigger a `click` event

**But**, since around 2015, both mobile Chrome and mobile Safari check for this:

https://github.com/umts/umasstransit.rodeo/pull/91/files#diff-a68281def00c6c6e8ca731b5f1b1727fR5

And if it's there, the assumption is that the user is not double-tapping to zoom since we promise zooming isn't required. So both browsers dispatch the `click` right away.

Reply:
Where this technique _really_ falls down is accessibility. I'm not really sure how to make this control keyboard-accessible (it might be possible, though). So, as is, keyboards and assistive devices aren't going to work very well for maneuver scoring.